invenio_rdm_records/resources/serializers/ui/schema.py

```diff
@@ -73,8 +74,8 @@ def apply_idx(affiliation):
         if 'role' in creator:
             creator['role']['title'] = gettext_from_dict(
                 creator['role']['title'],
-                get_locale(),
-                current_default_locale
+                locale=get_locale(),
```

Comment:
Function signature is without keyword arguments: ``def gettext_from_dict(catalog, locale, default_locale):``
```suggestion
                get_locale(),
```